### PR TITLE
Update DAS status-CR.include

### PR DESCRIPTION
**Pull request by marcoscaceres**
Joint deliverables with web apps wg should be done through GitHub, not mailing list.

### CODE REVIEW

**Comment by himorin** [comment]
@marcoscaceres you also need to remove part starting from line 31 (I suppose/believe feedback section in document detail is enough, and I saw some specs without this block, so just purging from joint deliverables should be fine)
Also we need to secure agreements from both WGs for joint deliverables - /cc @LJWatson @reillyeon @anssiko

**Comment by marcoscaceres** [comment]
Web Apps Charter already states, in the [working mode](https://www.w3.org/2022/04/webapps-wg-charter.html#working-mode):

> This group primarily conducts its technical work on GitHub. 

We explicitly removed using the mailing list for our work. If people were sending things the mailing list, it would get extremely messy (and no one does that). 

Additionally, DAS could even drop the mailing list requirement. Looking at the archive, no one has used the mailing list for technical discussion in literally years (a tiny bit in 2021 and basically not much going back before that):
https://lists.w3.org/Archives/Public/public-device-apis/

**Comment by reillyeon** [comment]
I'd prefer we swap out this mailing list text for something like, "If you wish to make comments regarding this document, please _file an issue on the specification repository_." where the italicized text is a link to the GitHub issues page.

**Comment by tabatkins** [comment]
And note you can use the `[REPOSITORY]` macro to make that link automatically work (especially when the `Repository` metadata is set, to avoid forks getting the wrong path).

**Comment by marcoscaceres** [comment]
Ok, great suggestions. Will update.

**Comment by anssiko** [comment]
@reillyeon https://webaudio.github.io/web-audio-api/ has this text:

>If you wish to make comments regarding this document, please [file an issue on the specification repository](https://github.com/WebAudio/web-audio-api/issues/new) or send them to [public-audio@w3.org](mailto:public-audio@w3.org) ([subscribe](mailto:public-audio-request@w3.org?subject=subscribe), [archives](https://lists.w3.org/Archives/Public/public-audio/)).

You probably looked at some other spec?

I think a good approach to cover all the bases would be to have the "Feedback:" header point to the singular GH repo issues list and keep the standard SOTD text per our earlier agreement, but with both the groups. This assuming we have the freedom to modify the "Feedback:" part without making checkers unhappy, and bikeshed supports that without hacks.

We should not avoid mentioning the mailing lists. We should instead make it clearer that GH is the preferred way to submit feedback.

**Comment by marcoscaceres** [comment]
@anssiko, I'm not sure what you are suggesting. 

Consider, for the joint deliverables we definitely don't want people sending feedback to two separate mailing lists. People in Web Apps don't monitor the DAS list, so it wouldn't make much sense to send feedback there - as they would just get redirected (by you and @reillyeon) to send their feedback to spec's repo regardless. 

Similarly, anyone not in WebApps from DAS wouldn't get to see the emails. 

It's fine for DAS to continue to promote the mailing list for their own specs, but listing the mailing lists (which are not used for technical discussions regardless) wouldn't make sense. 

> keep the standard SOTD text per our earlier agreement, but with both the groups. 

Can you remind me what we agreed to? I honestly don't recall... but pointing to DAS's mailing list for joint deliverables seems inappropriate. We could point to BOTH mailing list archives, but just not in the context of "send feedback" there.

Make sense?

**Comment by marcoscaceres** [comment]
Integrated the feedback. How does that look?

**Comment by anssiko** [comment]
@marcoscaceres if you have build this can you paste here a render for us to review?

In my mind, if we can, and are allowed to (pass checkers et al.), we'd have this line:

**History:**
...
**Feedback:**
please <a href="[REPOSITORYURL]/issues/new">file an issue on the specification repository</a>

... in the top matter of the spec. And then we could keep the mailing lists in the SOTD. This PR proposes to modify the SOTD. The Feedback customization we'd do IIRC by supplying custom metadata keys with a ! prefix in Bikeshed, maybe the same for ReSpec.

**Comment by marcoscaceres** [comment]
I don’t know how to generate this without first merging it, sorry (my bikeshed foo is limited… but I encourage you to do it). 

We are not changing the top/header of the document. That remains the same so no risk of affronting pub rules.

**Comment by marcoscaceres** [comment]
(Or rather, to change the Feedback:, that’s a different file). 

> And then we could keep the mailing lists in the SOTD. 

this keeps the mailing lists in the SoTD.

**Comment by marcoscaceres** [comment]
I’m having a hard time parsing what you wrote. Can you please do suggestions directly in the template?

**Comment by tabatkins** [comment]
> I don’t know how to generate this without first merging it, sorry (my bikeshed foo is limited… but I encourage you to do it).

If you put all the include files locally next to your spec file, Bikeshed will pick them up first rather than looking in its boilerplate folder.

**Comment by marcoscaceres** [comment]
@anssiko, here's the screenshot: 

Screenshot 2024-05-30 at 2 25 59 PM

**Comment by marcoscaceres** [comment]
This updated template: 

 * Leaves DAS specs as they are.  
 * includes @reillyeon suggested text.
 * Provides links to mailing lists and archives, as @anssiko requested, but does not suggest sending technical comments there (as WebApps working mode requires). 

That should address everyone's feedback.

File: boilerplate/dap/status-CR.include [hunk 1]
```diff
@@ -79,6 +77,16 @@
 	with <a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure">section 6 of the W3C Patent Policy</a>. 
 </p>
 
+<p include-if="Text Macro: JOINTWEBAPPS">
+  For general working group annoucements, admistrivial, and non-technical matters
```
Comment (anssiko) [inline boilerplate/dap/status-CR.include:81]:
```suggestion
  For general working group annoucements, administrative, and non-technical matters
```

**Comment by anssiko** [comment]
@marcoscaceres thanks for working through the Bikeshed magic. The screenshot looks good to me, one type fix.

File: boilerplate/dap/status-CR.include (hunk 1 above)
Comment (reillyeon) [reply boilerplate/dap/status-CR.include:81]:
```suggestion
  For general working group announcements, administrivia, and non-technical matters
```

**Review by reillyeon — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Ok, thanks. I’ll update the others.

**Comment by tabatkins** [comment]
Btw, please let me know when these are ready to merge.

**Comment by himorin** [comment]
@marcoscaceres @reillyeon @anssiko @LJWatson 
could you all give final go (or comment) on this? 
generated SoTD should be fine (not really tested w/pubrules - I haven't tried locally), but I believe final go need to be confirmed by chairs (and editors).

(I believe other five PRs will be updated per this change after this merged?)

**Review by reillyeon — APPROVED** [review]

**Pull request merged**